desk/lib/curios.hoon

```diff
@@ -34,18 +34,21 @@
   (need (get time))
 ::
 ++  reduce
-  |=  [=time del=delta:curios:h]
+  |=  [now=time =time del=delta:curios:h]
   ^+  cur
   ?-  -.del
+    :: we use now on the add to enforce host ordering
+    :: any other actions should use time so that it's
+    :: the note that we're looking for.
```

Comment:
This will use now on the add for both publisher and subscriber, so this comment is wrong.